datumaro/components/importer.py

```diff
@@ -181,6 +181,10 @@ def _find_sources_recursive(
                     break
         return sources
 
+    @classmethod
+    def get_extractor_name(cls) -> str:
+        return cls.NAME.replace("_importer", "")
```

Comment:
This seems like the duplication of https://github.com/openvinotoolkit/datumaro/pull/824.